### PR TITLE
Increasing resources for the tests.

### PR DESCRIPTION
**Pull request by MathieuDutSik**
## Motivation

The recent switch to the LTO introduced some problems with the CI.

## Proposal

Following proposals:
* Increasing the runtime to 60 minutes for the long_faucet. It already uses a beefy machine.
* Switch to a beefier machine for the `benchmark.yml`.

## Test Plan

The CI reliability is the issue here.

## Release Plan

Not relevant to TestNet nor DevNet.

## Links

None.

### CODE REVIEW

File: .github/workflows/benchmarks.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ env:
```
Comment (ndr-ds) [inline .github/workflows/benchmarks.yml:11]: <i class='graphite__hidden'>[Re: line +6]<br/><br/></i>

Could you send a temporary commit here enabling these on PRs so we can make sure they're passing before merging? Then you can remove the commit before merging. I think something like this here would work:

```gt_suggestion
  pull_request:
    branches:
      - "**"
  workflow_dispatch:
```

Same for the other test below<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/3018?utm_source=unchanged-line-comment">Graphite</a>.</span>

Comment (MathieuDutSik) [reply .github/workflows/benchmarks.yml:21]: Ok, done.

File: .github/workflows/long_faucet_chain_test.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ env:
 jobs:
```
Comment (ndr-ds) [inline .github/workflows/long_faucet_chain_test.yml:16]: <i class='graphite__hidden'>[Re: line +6]<br/><br/></i>

You need to do it for this one too<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/3018?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Comment by MathieuDutSik** [comment]
I think this PR is obsolete after the switch to "thin" for the LTO compilation.

**Pull request closed without merging**